docker-compose.yml

```diff
@@ -103,25 +44,20 @@ services:
     build:
       context: .
       dockerfile: pgbouncer/Dockerfile
-    image: "pgbouncer:${GIT_HASH}"
-    environment:
-      DEPLOYMENT_NAME: "DOCKER_TEST"
-
-  proxy:
-    build: proxy
-    image: "proxy:${GIT_HASH}"
-    environment:
-      DEPLOYMENT_NAME: "DOCKER_TEST"
-    ports:
-      - "80:80"
-    depends_on:
-      - app
+    image: pgbouncer
+  #proxy:
```

Comment:
we need to leave proxy in - even though CI doesn't use it, it needs to be built so it can be pushed to AWS